### PR TITLE
Uncontroversial updates

### PR DESCRIPTION
**Pull request by LawrenceHecht**
Making changes to the survey based on Steering Committee comments. I don't believe anything in this update is controversial

### CODE REVIEW

File: 2020/questions.md [hunk 1]
```diff
@@ -100,8 +102,10 @@ NEW QUESTION
 * IT
 * Office of the CTO
 * Developer relations, marketing or communications
+* Security and risk compliance
```
Comment (jeffmcaffer) [inline 2020/questions.md:105]:
```suggestion
* Security, compliance, risk management
```

"risk compliance" feels off

File: 2020/questions.md [hunk 2]
```diff
@@ -331,22 +337,19 @@ NEW question
...
 ### Part 5a: Value of Companies and Foundations
-32. To what degree do you perceive the following companies to be “good open source community citizens” in terms of their contributions, collaboration and leadership on open source projects and initiatives within the open source ecosystem? 
+32. "The following list represents a range of large companies that participate in open source communities. To what degree do you perceive each of them to be “good open source community citizens” in terms of contributions, collaboration and leadership on open source projects and initiatives within the open source ecosystem?"
+To what degree do you perceive the following companies to be “good open source community citizens” in terms of their contributions, collaboration and leadership on open source projects and initiatives within the open source ecosystem? 
 >{Excellent Above Average Average Below Average Very Poor Don't know}
 * AWS
+* Comcast 
+* Facebook
 * Google
-* Microsoft
```
Comment (jeffmcaffer) [inline 2020/questions.md:338]: Microsoft should stay in list. Logic for removing?

Comment (LawrenceHecht) [reply 2020/questions.md:338]: Microsoft was kept in -- for some reason, I don't know why it looks this way in the GitHub before vs after comparsion.

File: 2020/questions.md (hunk 1 above)
Comment (LawrenceHecht) [reply 2020/questions.md:105]: You're right. My bad. Last year, in "Other", "Security and Risk Management" and "Information Security/GRC"

File: 2020/questions.md [hunk 3]
```diff
@@ -59,22 +59,24 @@ NEW choices about employee contributions
...
-* Somewhat likely
+
+7. In light of recent macroeconomic conditions, what is the likelihood that funding for your company's open source initiatives will increase or decrease in the upcoming fiscal year?
+* Very likely to increase
+* Somewhat likely to increase
 * Neutral
-* Somewhat unlikely
-* Very unlikely
+* Somewhat likely to decrease
+* Very likely toi decrease
```
Comment (DuaneOBrien) [inline 2020/questions.md:79]: 'toi' is a typo

**Pull request merged**